test/mocha/mock.data.js

```diff
@@ -1,72 +1,70 @@
+/* eslint-disable max-len */
 /*!
 * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
 */
 'use strict';
 
-const {config: {constants}} = require('bedrock');
+const {constants: {VERES_ONE_CONTEXT_V1_URL}} = require('veres-one-context');
+const {constants: {DID_CONTEXT_URL}} = require('did-context');
+const {constants: {CREDENTIALS_CONTEXT_V1_URL}} = require('credentials-context');
```

Comment:
+1 to what you've done here as a step in the right direction.  In the near future, hopefully we will have the constants exported by their `bedrock-foo-context` counterparts.  This will allow us to eliminate direct dependencies on the context modules.  We already have dependencies on the `bedrock-foo-context` counterparts.